misc/ls-extensions/modules/project-design-service/src/main/java/io/ballerina/projectdesign/model/entity/Attribute.java

```diff
@@ -35,8 +38,8 @@ public class Attribute {
     private final List<Association> associations; // can have multiple association when union is found
 
     public Attribute(String name, String type, boolean optional, boolean nillable, String defaultValue,
-                     List<Association> associations) {
-
+                     List<Association> associations, ModelElementLocation elementLocation) {
```

Comment:
I'd also prefer the name `ElementLocation` or `SourceLocation` for the newly added location class :)

```suggestion
                     List<Association> associations, ElementLocation elementLocation) {
```

Reply:
Updated as `ElementLocation`